### PR TITLE
Main skeleton

### PR DESCRIPTION
**Pull request by KCnick**
Implemented a clone of the Newsweek website using bootstrap.

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+  padding-right: 0;
+}
+#my-turn .card-body,
+#culture-travel .card-body {
+  color: #999;
+  font-weight: bold;
+}
+#culture-travel .card-text {
+  padding: 2px 5px !important;
+  color: #333;
```
Comment (bertil291utn) [inline assets/css/main.css]: ## No `!important` overusing[1]

The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. 

- [ ] Remove all `!important` rules in your entire CSS file

<hr>

**Note:** Check all this entire page and fix them

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: assets/css/main.css [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+}
+
+/* NEWS */
+
+#news {
+  margin-top: 60px;
+}
+#news h2 {
+  color: #999;
+  text-transform: uppercase;
```
Comment (bertil291utn) [inline assets/css/main.css]: ## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color: #999;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.color-x{
  color: #999;
}
```
**Html file**

```HTML
<div class="color-x">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: assets/css/main.css [hunk 3]
```diff
@@ -0,0 +1,516 @@
...
+  background-color: #ffe400;
+  border-radius: 99em;
+  padding: 1px 10px;
+  white-space: nowrap;
+}
+#hamburger {
+  font-size: 1em;
+  color: #fff;
+  margin-left: 0.5em;
+}
```
Comment (bertil291utn) [inline assets/css/main.css]: ## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `  color: #fff;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.white-color{
    color: #fff;
}
```
**Html file**

```HTML
<div class="white-color">
```
**Note:** Do this change to all properties in this file.
- Also for join `color: white;` statement in this same class

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, congratulations 🎉 guys is a wonder full job, this page looks amazing
, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

### General requirements
- Fix linter issues, related style lint, and webhint 
- ["In the magazine"](https://user-images.githubusercontent.com/24902525/101646028-077fde00-3a05-11eb-90b9-618a47c3819b.png) section is missing, please add it 
### FILE: README.md
- Use this [mockup](https://www.mockupworld.co/free/responsive-web-design-showcase-mockup/) to add a screenshot 
- If you don't; have something to add to the acknowledgment section, please remove it, although it'd be great to add https://www.newsweek.com  as inspiration or helpful web page 

### Specific requirements

| Section/Screen                             | Large | Medium | Small |
| :----------------------------------------- | :---: | :----: | :---: |
| Navbar              |      |      |       |
| Featured stories              |      |        |       |
| Top story             |       |        |       |
| My turn             |       |        |       |
| Cultural&Travel             |      |        |       |
| Opinion                 |       |        |       |
| Latest news                  |       |        |       |
| More stories |      |        |       |
| The debate |      |    1    |       |
| In the magazine         |       |        |       |
| Editor's pick |    |   2  |      |
| Footer                                     |       |        |       |
###### This table is given each section with his screen width approach, each issue is given by number below this table

1. Move to the left this image, according to the original page 
image

2.  It has to be only two rows for medium screens 

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

3. Please this order for small screens approach, order your web page to have the same approach as the original one 
**This is the correct order**
   - Top story
   - Featured stories
   - The debate
   - Opinion 
   - More Stories
   - My Turn
   -  Culture& Travel
   - In the magazine
   - Editors pick
   - Footer

image

### Summary
- Congratulations 🎉 `guys is a wonder full job, this page looks amazing
- Add the missing page section 
- Work on CSS good practices (check inline comments for more information)
-  Don't forget to work with inline comments below this comment box.

***

Guys work on the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @KCnick  , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**